addons/core/functions/fnc_activeSwRadio.sqf

```diff
@@ -24,4 +24,9 @@ private _result = nil;
     if (_x call TFAR_fnc_isRadio) exitWith {_result = _x};
     true;
 } count (assignedItems TFAR_currentUnit);
+
+if (player != TFAR_currentUnit && {TFAR_remoteRadio}) then {
+    _result = TFAR_remoteRadioItem;
```

Comment:
This is very suboptimal. We scan all the Items on the player. Only to then find out we don't need that info.
Optimally move this if before the loop and `exitWith {TFAR_removeRadioItem}`